src/pat/validation/tests.js

```diff
@@ -27,6 +27,24 @@ define(["pat-registry", "pat-validation"], function(registry, pattern) {
             expect($el.find('em.warning').length).toBe(0);
         });
 
+        it("validates required pat-autosuggest inputs on form submit", function() {
+            var $el = $(
+                '<form class="pat-validation">'+
+                '<input type="hidden" name="name" required="required" class="pat-autosuggest" data-pat-autosuggest="words: one, two">'+
+                '</form>');
+            pattern.init($el);
+            $el.submit();
+            expect($el.find('em.warning').length).toBe(1);
+
+            // Check that validation passes when required is not here.
+            $el = $(
+                '<form class="pat-validation">'+
+                '<input type="hidden" name="name" class="pat-autosuggest" data-pat-autosuggest="words: one, two">'+
+                '</form>');
+            $el.submit();
```

Comment:
Do we need another `pattern.init($el)` here?

Reply:
good catch, yes it does. checked it and added it myself.

Reply:
If yes then we probably also need it in the previous test which I copied this from...

Reply:
perfect, also added it there.